docs/content/operations/cli.md

```diff
@@ -26,6 +26,8 @@ traefik [--flag=flag_argument] [-f [flag_argument]]
 traefik [--flag[=true|false| ]] [-f [true|false| ]]
 ```
 
+Flags are case insensitive.
```

Comment:
```suggestion
!!! info "Flags are case insensitive."
```

Reply:
Goal of the suggestion: put a light emphasis on this sentence.

Rendering:

<img width="729" alt="Screenshot 2019-09-19 at 12 22 41" src="https://user-images.githubusercontent.com/1522731/65236129-3811ea00-dad8-11e9-8380-056d90561945.png">